app/src/main/java/com/github/se/project/ui/lesson/AddLesson.kt

```diff
@@ -0,0 +1,239 @@
+            modifier = Modifier.padding(16.dp))
+      },
+      content = { paddingValues ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(16.dp).padding(paddingValues),
+            verticalArrangement = Arrangement.spacedBy(8.dp)) {
+              OutlinedTextField(
+                  value = title,
+                  onValueChange = { title = it },
+                  label = { Text("You can write what the lesson is about in short") },
+                  placeholder = { Text("Give a title to this lesson") },
+                  modifier = Modifier.fillMaxWidth(),
+                  singleLine = true)
+
+              // TODO: description picture
+
+              OutlinedTextField(
+                  value = description,
+                  onValueChange = { description = it },
+                  label = { Text("You can write what the lesson is about in detail") },
+                  placeholder = { Text("Give a description to this lesson") },
+                  modifier = Modifier.fillMaxWidth(),
+                  singleLine = true)
+
+              Text("Select the desired date and time for the lesson")
+              Button(onClick = { datePickerDialog.show() }) {
+                Text(if (selectedDate.isEmpty()) "Select Date" else "Selected Date: $selectedDate")
+              }
+
+              Button(onClick = { timePickerDialog.show() }) {
+                Text(if (selectedTime.isEmpty()) "Select Time" else "Selected Time: $selectedTime")
+              }
+
+              // Subject input
+              WritableDropdown(
+                  label = "Which subject would you like to study ?",
+                  placeholder = "Subject(s)",
+                  value = subject,
+                  onValueChange = { subject = it },
+                  choices = TutoringSubject.entries.map { it.name }.toList())
+
+              // Language input
+              WritableDropdown(
+                  label = "In which language would you like the lesson to take place ?",
+                  placeholder = "Language(s)",
+                  value = language,
+                  onValueChange = { language = it },
+                  choices = Language.entries.map { it.name }.toList())
+
+              // Slider price selection
+              PriceSlider("Select a price range for your lesson:") { min, max ->
+                minPrice = min.toDouble()
+                maxPrice = max.toDouble()
+              }
+              Text("Selected price range: ${minPrice.toInt()}.- to ${maxPrice.toInt()}.-")
+            }
+      },
+      bottomBar = {
+        Button(modifier = Modifier.fillMaxWidth().padding(16.dp), onClick = onConfirm) {
+          Text("Confirm your details")
+        }
+      })
+}
+
+/** Return a potential validation error */
+fun validateLessonInput(
+    title: String,
+    description: String,
+    subject: String,
+    language: String,
+    minPrice: Double,
+    maxPrice: Double,
```

Comment:
here, minPrice and maxPrice variables are not used in the logic, you can just remove them since they have default values

Reply:
Done